src/tactic/ui/panel/tool_layout_wdg.py

```diff
@@ -29,6 +29,32 @@ class ToolLayoutWdg(FastTableLayoutWdg):
             'order': 0,
             'category': 'Required'
         },
+         "search_limit_mode": {
+            'description': "Determine whether to show the simple search limit at just top, bottom, or both'",
+            'type': 'TextWdg',
+            'order': 1,
+            'category': 'Display'
+        },
+        "expand_mode" : {
+            'description': 'support Tile Layout gallery, single_gallery, plain, detail, and custom mode',
```

Comment:
Capitalize the first word of description for all of them, order should be 0, 1, 2, 3, 4